### DISCUSSION TITLE
Add Note in new GitHub

### DISCUSSION THREAD

**Discussion by donnabu** [community/community#132520] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

In the previous version of GitHub, there as an option to add a Note in the Project that was always displayed.  This was very helpful to easily see important dates and information about a release.  However, this option does not seem to be available in the New GitHub Project view.  Is there a way to add a note that is always open in the Project view?

**Comment by harshjuly12** [comment]
Hey @donnabu you can use the project description field for imp notes and info or pinned items field for imp issues and pull requests.

**Comment by donnabu** [comment]
Thank you @harshjuly12 for the quick response.  I appreciate the information; however, it seems the "Short description" option only allows sentence structure which is not easy to read for a list of important dates.  Am I correct that you cannot added a formatted bulleted list in the description?

**Reply by donnabu** [reply]
I found a workaround by adding the important dates bulleted list in the project "Readme" section.

**Reply by harshjuly12** [reply]
Yes actually short description section doesn't support formatted text like bulleted lists. It is designed for a plain-text summary of the repository only. Also that's an awesome solution to format important dates in the README.md file this way it will be more readable.
